mets.md

```diff
@@ -160,6 +199,83 @@ encodings of the same page.
 </mets:structMap>
 ```
 
+## OCR-D structMap
+
+A METS can contain different `structMap`. These are defined by the `TYPE` attribute (e.g. `LOGICAL`, `PHYSICAL`>) described in more detail.	
+A document structure determined by OCR-D is described by `TYPE="OCR-D-LOGICAL"`.
+
+attributes in `structMap` | description
+--                        | --
+`LABEL`                   | contains the character recognition result  e.g. the title of the chapter
+`TYPE`                    | contains a standardized description of structur (see.: [DFG-Viewer: structural data set](https://dfg-viewer.de/strukturdatenset/))
```

Comment:
```suggestion
`TYPE`                    | contains the type of a structuring element according to a standardized, controlled vocabulary (see.: [DFG-Viewer: structural data set](https://dfg-viewer.de/strukturdatenset/))
```

Reply:
I think the the proposal is not better.
element -> is ambiguous, and it is not an element it is an index term or subject term or descriptor
For this reason I propose:
`contains a descriptor according to a standardized, controlled vocabulary`

Reply:
My formulation doesn't just use _element_ but the compound noun _structuring element_. Both rows refer to this in my suggestion (i.e. its _text_ and its _type_). A heading or caption is of course a structuring element (within a structured document), not an `index term or subject term or descriptor`.

Reply:
No, the value of `@TYPE` is an **descriptor**.

Reply:
> No, the value of `@TYPE` is an **descriptor**.

I get a feeling this is just a misunderstanding:

Of course `@TYPE` is a descriptor (as is `@LABEL`). But they **belong** to (or refer to) a _structuring element_ like `chapter`. You'll have to explain the general thing first, however obvious it may seem. In contrast, that `@TYPE` is a descriptor is not worth extra mention IMO.

See what I mean?

Reply:
I see it only now. Here is missing a description to  'mets:div'.

Reply:
> Here is missing a description to 'mets:div'.

Yes, you _could_ point that out in the table header – for example by replacing
> attributes in `structMap`

with
> attributes of `mets:structMap/mets:div/mets:div`

But this is _not_ what I meant...

Reply:
> > attributes of `mets:structMap/mets:div/mets:div`
> 
> But this is _not_ what I meant...

So we agreed that the recursive nature  of these `mets:div` should be mentioned as well.

But the culprit here was that @tboenig is concerned the term _element_ gets confused with its technical reading in the context of the XML standard.

But since we somehow still have to say just what a structMap consists of (i.e. what `@TYPE` and `@LABEL` describe), I proposed to use the term _(structuring) component_ instead.

This would become:
```suggestion
`TYPE` | contains the type of a structuring component according to some standardized, controlled vocabulary (see.: [DFG-Viewer: structural data set](https://dfg-viewer.de/strukturdatenset/))
```